cloudify_aws/ec2/instance.py

```diff
@@ -405,6 +406,13 @@ def _handle_userdata(self, parameters):
             final_userdata = compute.create_multi_mimetype_userdata(
                     [existing_userdata, install_agent_userdata])
 
+        if install_agent_userdata and os_family == 'windows':
+            if not final_userdata.startswith('<powershell>') and not \
+                    final_userdata.endswith('</powershell>'):
+                final_userdata = \
+                    '<powershell>\n{0}\n</powershell>'.format(
+                        final_userdata)
+
```

Comment:
That won't work. You shouldn't surround everything by `<powershell>` because that assumes that all of the userdata content (including the one provided by the user) is also PowerShell.